### ISSUE TITLE
cant see the pre-input message in dota2 at the simple-chinese runtime-environment

### ISSUE THREAD

**Issue by yfh349958021** [jeffhill/Dota2#604]
### Description

in simple-China area,we cant see the pre-input message in dota2 game with Microsoft InputMethod-chinese,and Microsoft InputMethod-chinese work in other place 
![QQ截图20220605222847](https://user-images.githubusercontent.com/19186834/172055787-2874c778-c5da-4a5e-86fd-af6b82403d98.png)
![QQ截图20220605221723](https://user-images.githubusercontent.com/19186834/172055836-69fd3531-9c45-4d31-88b5-5d31f43e527b.png)

**Comment by AmanovAkbar** [comment]
That's true. It's simply infuriating when I try to find the correct character blindly, when texting to my Chinese teammates. You see, in "Microsoft input method - Chinese" you type in pinyin(Latin alphabet letters) and then a list of options for a character is shown like on the screenshot above and you use numbers on a keypad to choose the correct option. The problem is that in Dota that list of options is not shown, and you really gotta guess which number has the character you need and if you are mistaken, you have to type the character again and hope that this time you guess correctly. What is also annoying is that the arrangement of those characters might change due to frequency of use and stuff so it makes guessing process even harder. What is funny is that said list of options is perfectly visible in steam layout. 
Please, valve, fix this bug. It's frustrating and embarrassing when instead of explaining your opinion on current in-game situation you end up saying that you grassed someone's horse or smth.
